backend/api/routers/pool_router.py

```
@@ -213,4 +217,86 @@ async def import_pool(
+        return arc_stats
+    except SecurityValidationError as e:
+        raise HTTPException(status_code=422, detail=f"Security validation failed: {str(e)}")
+    except Exception as e:
+        logger.error(f"Error getting ARC stats: {e}")
+        raise HTTPException(status_code=500, detail=str(e))
+
+
+@router.post("/{pool_name}/scrub/start", response_model=Dict[str, Any])
+async def start_pool_scrub(
+    pool_name: str,
+    pool_service: PoolService = Depends(get_pool_service)
```

Ruff (0.11.9)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)